tests/pyximport/pyximport_pythonpath.srctree

```diff
@@ -0,0 +1,21 @@
+PYTHON -c "import pyximport_test; pyximport_test.test()"
```

Comment:
It might be worth combining the two srctree tests (the pyx and py modules would need to have different names but that's all)

Reply:
Tests merged together.